docs/navigate.md

```diff
@@ -57,6 +60,20 @@ update itself while the user is panning around on it (set to null to disable
 intermediate pans, the plot will then not update until the mouse button is
 released).
 
+**mode** a string specifies the pan mode for mouse interaction. Accepted values:
+empty string for normal mode: no pan hint or direction snapping;
+'smart': The graph shows pan hint bar and the pan movement will snap
+to one direction when the drag direction is close to it;
+'smartLock'. The graph shows pan hint bar and the pan movement will always
+snap to a direction that the drag diorection started with.
+Default: 'smart'.
+
+**enableTouch** enables the touch support, including pan (to drag), pinch (to zoom and move),
+and double tap (to recenter).
+
+**touchMode** a string specifies the pan mode of touch pan.
```

Comment:
I still don't know why we're introducing a unique property for the touch case. I'm pretty sure this plugin is meant to be used side-by-side with the regular navigate plugin, which will already introduce the 'mode' property. I find it hard to believe for situations where the user can interact with the same graph through both touch and the mouse, that they want different pan modes.